docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+services:
+  dick:
+    container_name: dick
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    command: npm start
+    volumes:
+      - ./constants.ts:/app/dick/src/constants.ts # DICK constants file
```

Comment:
Should be `- ./src/constants.ts:/app/dick/src/constants.ts # DICK constants file